.github/test-spec.yml

```diff
@@ -97,7 +99,10 @@
   - any:
       - "include/zephyr/bluetooth/**/*"
       - "!include/zephyr/bluetooth/mesh/**/*"
-  - "samples/bluetooth/hci_ipc/**/*"
+  - any:
+      - "boards/nordic/nrf5*"
+  - any:
+      - "soc/nordic/**/*"
 
```

Comment:
Do we want to enable "subsys pm" to trigger BLE CI as well if the PM related configuration broke BLE CI this time?